lib/rubocop/cop/rails/find_by.rb

```diff
@@ -45,6 +46,15 @@ def autocorrect(node)
             corrector.replace(first_loc, '')
           end
         end
+
+        private
+
+        def where_with_method_as_param?(node)
+          where = node.children.find { |el| el.method?(:where) }
```

Comment:
Hm. This makes the assumption that all children are `send` nodes (because only `SendNode` implements `#method`.

A safer way is:

```ruby
node.each_child_node(:send).find { |child| child.method?(:where) }
```

`#each_child_node` will return an iterator containing only nodes of the optional type.